src/t2iapi/operation/service.proto

```diff
@@ -29,4 +29,12 @@ service OperationService {
     rpc SetOperatingMode (t2iapi.operation.SetOperatingModeRequest)
         returns (BasicResponse);
 
+    /*
+    Set the @InvocationEffectiveTimeout of the pm:AbstractOperationDescriptor with the given handle to a value that is
+    less than or equal to the requested threshold.
+    The manipulated descriptor shall be persistent until a next manipulation call or an SDC operation invocation. If
+    the device is not able to maintain the static descriptor, it shall return RESULT_NOT_SUPPORTED.
+    */
+    rpc SetInvocationEffectiveTimeoutLteToThreshold (SetInvocationEffectiveTimeoutLteToThresholdRequest)
```

Comment:
The name of the rpc seems strange SetInvocationEffectiveTimeoutLteToThreshold what means Lte??

Reply:
yeah this was a suggestion to shorten the name of the manipulation
lte is the abbreviation of LessThanOrEqual

but your are right, it leaves room for misinterpretation, i have written it in the full length, now